src/components/WeatherWidget.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div class="weather-widget">
+    <WeatherWidgetPlace
+      :="weatherData.location"
+      :date="selectedDayData.date ?? 'Today'"
+    />
+    <WeatherWidgetCurrentData :="selectedDayData" />
+    <WeatherWidgetForecast
+      :forecastData="weatherData.forecast"
+      :selectedDay="selectedDay"
+      @handleDayChange="handleDayChange"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import WeatherWidgetPlace from "./WeatherWidgetPlace.vue";
+import WeatherWidgetCurrentData from "./WeatherWidgetCurrentData.vue";
+import WeatherWidgetForecast from "./WeatherWidgetForecast.vue";
+import { computed, ref } from "vue";
+import { mockWeatherResponseData } from "../helpers/mockWeatherDataResponse";
+import { mapWeatherData } from "../utils.ts/dataMappers";
+
+const selectedDay = ref<number | null>(null);
```

Comment:
selectedDay should be the actual day (e.g. 2019-07-07) since indices tend to change and they make it harder to understand what exact date is currently selected.

Reply:
Resolved.